providers/diff/diff.go

```diff
@@ -221,6 +221,32 @@ func (d *differ) ChangedGroups(existing []*models.RecordConfig) map[models.Recor
 	return changedKeys
 }
 
+// DebugKeyMapMap debug prints the results from ChangedGroups.
+func DebugKeyMapMap(note string, m map[models.RecordKey][]string) {
```

Comment:
This looks like debug code that should maybe be removed. The only usage is commented out.

Reply:
This was originally I found this useful enough that I moved it to the diff package.  I've made it activate in gandi_v5 if the debug flag is true.